packages/manager/modules/product-offers/src/product-offers.component.js

```diff
@@ -0,0 +1,87 @@
+import controller from './product-offers.controller';
+import template from './product-offers.html';
+
+/**
+ *  Product offers component. Can use several workflows. See details of each
+ *  workflow in the concerned folder.
+ *
+ *  The component is based on ovh-ui-kit 'oui-stepper'
+ *  It is aimed to handle the following order steps:
+ *  - Pricing choice
+ *  - Payment
+ *
+ *  This can be used in combination with a configuration step, if more details
```

Comment:
Only a configuration step can be added ? What if we want to add several steps because it is required to do so ?

Reply:
You can have multiple steps before, so I'll change the phrase